client/src/components/Home/mainPage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import styles from "./styles.module.scss";
+
+export const MainPage = () => {
+	return (
+		<main>
+			<div className={styles.board}>
+				<span className={styles.empty}>
+					This board is empty. Create a new column to get started
```

Comment:
Button is being created in a different task https://github.com/users/Erikang20/projects/2/views/1?pane=issue&itemId=68729484